src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -267,6 +268,9 @@ export default class ResultsViewPage extends React.Component<
                             <Mutations
                                 store={store}
                                 appStore={this.props.appStore}
+                                sampleIdToClinicalDataMap={
+                                    store.clinicalDataGroupedBySampleMap.result
```

Comment:
strange that we pass in store (269) and then one of it's properties.    also, when you pass in the result like this, you can no longer handle it's pending/isComplete status inside.  Make sure that we are properly handing the loading state.

Reply:
right, we could handle this inside `Mutations.tsx`